### PR TITLE
Support ocr box stitching

### PR DESCRIPTION
**Pull request by gaotongxiao**
New utils as suggested in #272
Any suggestions on variable naming & documentation are appreciated :) 
Todo: add demos to notebook

### CODE REVIEW

**Comment by innerlee** [comment]
How about a new file `box_util.py`

File: mmocr/utils/img_util.py [hunk 1]
```diff
@@ -49,3 +50,119 @@ def is_not_png(img_file):
...
+def is_on_same_line(box_a, box_b, min_y_overlap_rate=0.8):
+    """Check if two boxes are on the same line by their y-axis coordinates.
+
+    Two boxes are on the same line if they overlap vertically, and the length
+    of the overlapping line segment is greater than min_y_overlap_rate * height
+    of either of the boxes.
+
+    Args:
+        box_a (list), box_b (list): Two bounding boxes to be checked
+        min_y_overlap_rate (float): The minimum vertical overlapping rate
```
Comment (innerlee) [inline mmocr/utils/img_util.py:64]: what is a "rate"? do you mean "ratio"?

**Comment by gaotongxiao** [comment]
> How about a new file `box_util.py`

Or `postprocessing.py`?

**Comment by innerlee** [comment]
Can you test it?

Besides,

> How about a new file box_util.py

**Comment by gaotongxiao** [comment]
> Can you test it?
> 
> Besides,
> 
> > How about a new file box_util.py

Vote for `postprocess.py` though

**Review by innerlee — APPROVED** [review]

**Pull request merged**